forc-pkg/src/manifest.rs

```diff
@@ -127,6 +128,52 @@ impl Manifest {
     }
 }
 
+// Checks for unused manifest keys and displays a warning for each key present.
+//
+// This strategy is a combination of [cargo/util/toml/mod.rs](https://github.com/rust-lang/cargo/blob/489b66f2e458404a10d7824194d3ded94bc1f4e4/src/cargo/util/toml/mod.rs#L100),
+// [serde_ignored](https://docs.rs/serde_ignored/latest/serde_ignored/)'s default example and `forc-pkg::pkg::validate`.
+fn warn_unused_key(manifest: &str) -> anyhow::Result<()> {
+    let manifest_str = manifest;
```

Comment:
Rather than reassigning here, the function argument binding could be changed to `manifest_str: &str`

Reply:
ah thanks!